code/modules/mapping/map_template.dm

```diff
@@ -25,7 +26,11 @@
 			cached_map = parsed
 	return bounds
 
-/datum/parsed_map/proc/initTemplateBounds()
+/datum/parsed_map/proc/initTemplateBounds(datum/map_template/template)
+	// Check here before we have to go though all of this.  Its all done in SSatoms anyway
+	if(!SSatoms.initialized)
```

Comment:
The enum is never set other than 0 till SSatoms is run.  That was one of the pr's I did a while ago because the enum would be all over the place during map load.  Ugh let me check again, that was a few months ago.

Reply:
ok changed a few things about this so might need to review it again